saturn-r/postBuild

```diff
@@ -0,0 +1,53 @@
+# https://cran.rstudio.com/bin/linux/debian/
+sudo apt-key adv --keyserver keys.gnupg.net --recv-key 'E19F5F87128899B192B1A2C2AD5F960A256A04AF'
+
+sudo apt-get update
+sudo apt-get install -y \
+    --no-install-recommends \
+        software-properties-common
+
+sudo su -c \
+    "echo 'deb http://cloud.r-project.org/bin/linux/debian buster-cran40/' >>/etc/apt/sources.list"
+
+sudo apt update
+sudo apt install -y -t buster-cran40 \
+    r-base \
+    r-base-dev 
+
+sudo apt-get update
+sudo apt-get install -y \
```

Comment:
A few questions here:
1. Why is a second `update` needed after the one on line 12? I see that you need `-t` for the first `apt install` and not for the second, so I get why those need to be separate.
2. You've got mixed `apt` and `apt-get` usage here - why not just use `apt` everywhere?

Reply:
Good questions! Let me experiment and see what I can pare down- it might be relics from the development process, I can't remember offhand.

Reply:
Yeah the legacy of this image traces back to a customer sending us a script and then Stephanie and I trading off forking off and modifying that, so I can see how things got mixed! I think replacing all the `apt-get` with `apt` should work without issue.

Reply:
Ok, got that tidied up- how does it look now?

Reply:
@forana James approved but I'll leave this for a little longer today just to give you a chance at one more look, in case we missed anything.